src/test/playlisttest.cpp

```diff
@@ -56,3 +57,52 @@ TEST_F(PlaylistTest, Relative) {
     EXPECT_EQ(QString("base/folder/../../bar.mp3"),
             parser.playlistEntryToFilePath("../../bar.mp3", "base/folder"));
 }
+
+TEST_F(PlaylistTest, m3uEndOfLine) {
```

Comment:
Everywhere else we use PascalCase for the test names, let's keep this pattern here:

```suggestion
TEST_F(PlaylistTest, M3UEndOfLine) {
```